src/components/home/BannerCard/BannerCard.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import Typography from '../../common/Typography/Typography';
```

Comment:
Suggestion: No need to import React on every component.

Reply:
Eslint rule says React should be imported. Though, it can be disabled, but I prefer React to be imported. This is no anti-pattern.